docs/core/tools/global-tools-how-to-create.md

```diff
@@ -0,0 +1,201 @@
+---
+title: How to create a .NET Core Global Tool
+description: Describes how to create a Global Tool. The Global Tool is a console application that is installed through the .NET Core CLI.
+author: Thraka
+ms.author: adegeo
+ms.date: 08/22/2018
+---
+
+# Create a .NET Core Global Tool
+
+This article teaches you how to create and package a .NET Core Global Tool. The .NET Core CLI allows you to compile and mark a console application as a Global Tool, which others can easily install. .NET Core Global Tools are NuGet packages that are installed from the .NET Core CLI. For more information about Global Tools, see [.NET Core Global Tools overview][global-tool-info].
+
+[!INCLUDE [topic-appliesto-net-core-21plus.md](../../../includes/topic-appliesto-net-core-21plus.md)]
+
+## Create a project
+
+This article uses the .NET Core CLI to create and manage a project. Obviously you could use Visual Studio to do some of these steps, but Visual Studio will not be described in this article.
+
+First, create a new .NET Core 2.1 Console Application.
+
+```console
+dotnet new console -o botsay
+```
+
+Enter the `botsay` directory created by the previous command.
+
+In this example we'll create a console application that generates an ascii bot and prints a message. The first thing we'll do is use the same command line parsing system that makes it easy to parse arguments and provides a help system.
+
```

Comment:
Starting with "The first thing.." delete this and following - the rest of the section

Reply:
Whoops.. left over intro.